shared-helpers/src/views/sign-in/FormSignIn.tsx

```diff
@@ -34,66 +36,136 @@ const FormSignIn = ({
     window.scrollTo(0, 0)
   }
   const { LinkComponent } = useContext(NavigationContext)
-
   return (
-    <FormCard>
-      <div className="form-card__lead text-center">
-        <Icon size="2xl" symbol="profile" />
-        <h1 className="form-card__title">{t(`nav.signIn`)}</h1>
-      </div>
-      <FormSignInErrorBox
-        errors={errors}
-        networkStatus={networkStatus}
-        errorMessageId={"main-sign-in"}
-      />
-      <div className="form-card__group pt-0">
-        <Form id="sign-in" className="mt-10" onSubmit={handleSubmit(onSubmit, onError)}>
-          <Field
-            caps={true}
-            name="email"
-            label={t("t.email")}
-            validation={{ required: true }}
-            error={errors.email}
-            errorMessage={t("authentication.signIn.enterLoginEmail")}
-            register={register}
-            dataTestId="sign-in-email-field"
+    <Card>
+      {process.env.showMandatedAccounts ? (
+        <>
+          <CardHeader className="mx-12 space-y-3 pl-0 py-8">
+            <Icon size="2xl" symbol="profile" />
+            <Heading size="2xl" className="font-semibold">
+              {t("nav.signInLowercase")}
```

Comment:
@emilyjablonski Correct me if I'm wrong, but I was interpreting your comment about translations to mean that this should stay as "Sign In" until a full set of translations are ready for the switch to lowercase

Reply:
We can't release this feature without translations, so if we know we are getting translations before release we are good to add new strings - otherwise we shouldn't change any strings or add any.